<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProTra</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>stopwatch.png</normaloff>stopwatch.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>681</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="widget">
     <attribute name="icon">
      <iconset>
       <normaloff>timer.png</normaloff>timer.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Timer</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="3" colspan="2">
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="font">
         <font>
          <family>EADS Sans</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="4">
       <widget class="QTableWidget" name="tableWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>200</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>EADS Sans</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <row>
         <property name="text">
          <string>1</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Project</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Time on project</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>Project A</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="pushButtonExport">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Palatino Linotype</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>EXPORT</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>export.png</normaloff>export.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>EADS Sans</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="lineEdit">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>EADS Sans</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButtonStop">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>EADS Sans</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>STOP</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>stop.png</normaloff>stop.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButtonPause">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>EADS Sans</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>PAUSE</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>pause.png</normaloff>pause.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonStart">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>EADS Sans</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>START</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>start.png</normaloff>start.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButtonReset">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>EADS Sans</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>RESET</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>reset.png</normaloff>reset.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pushButtonCommit">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>EADS Sans</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>COMMIT</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>add.png</normaloff>add.png</iconset>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="pushButtonAdd">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>EADS Sans</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ADD ROW</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>add2.png</normaloff>add2.png</iconset>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="pushButtonRemove">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>EADS Sans</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>REMOVE ROW</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>remove.png</normaloff>remove.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget">
     <attribute name="icon">
      <iconset>
       <normaloff>plot-8.png</normaloff>plot-8.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Graphic</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>651</width>
        <height>451</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="plotlayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="leftMargin">
        <number>100</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>100</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>714</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tableWidget</sender>
   <signal>cellChanged(int,int)</signal>
   <receiver>comboBox</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
